### ISSUE TITLE
Support for docstrings

### ISSUE THREAD

**Issue by julianLapenna12** [johnsamuelwrites/inclusive#2]
I'm considering using a tool similar to this, but I would need it to work to automatically scan docstrings... maybe something that works in tandem with PyDocStyle? I'm not quite sure

**Comment by johnsamuelwrites** [comment]
@julianLapenna12 Thanks for creating this interesting issue. If I get this correctly, you would like to scan the code/programs using an extended version of [PyDocStyle](https://github.com/PyCQA/pydocstyle) that includes `inclusive` that can indicate non-inclusive sentences in the code.

**Comment by julianLapenna12** [comment]
Yes, it's not immediate and I'm also looking around for other libraries. But essentially on the larger scale, my team has been looking at a docstring linter that could detect words that are not inclusive and suggest replacements (i.e. `abort` would be flagged and suggested to be replaced with `force quit`).

**Comment by johnsamuelwrites** [comment]
This is an interesting use case: integration with docstring linter. `inclusive` is generic as well as configurable. New words in any language can be easily added to the list on the fly by the user. E.g., https://github.com/johnsamuelwrites/inclusive/blob/master/inclusivewriting/resources/en/list.json
